### ISSUE TITLE
Display the CVSS in suggestion data

### ISSUE THREAD

**Issue by fricklerhandwerk** [Nix-Security-WG/nix-security-tracker#290]
As a security team member, I want to assess the severity of a CVE at a glance, and be able to inspect it in more detail.

Depends on:
- https://github.com/Nix-Security-WG/nix-security-tracker/issues/330

**Comment by erictapen** [comment]
![desktop](https://github.com/user-attachments/assets/682204e9-e1d0-4e5e-8cc5-7967b1934e77)

@fricklerhandwerk and I did a rough sketch for it. The base score is shown next to the CVE id in the upper left hand corner. By clicking on it it unfolds into a table that displays the different components of the vector, including links to the documentation of the individual fields. For now we just screenshotted the explanations from [nvd.nist.gov](https://nvd.nist.gov/vuln/detail/CVE-2022-47597).
